addons/website_sale/views/templates.xml

```diff
@@ -1839,7 +1839,7 @@
     </template>
 
     <template id="row_addresses">
-        <div t-attf-class="{{'all_billing' if is_invoice else 'all_delivery'}} row row-cols-md-2 row-cols-lg-3 g-3 flex-nowrap flex-md-wrap mb32">
+        <div t-attf-class="{{'all_billing' if is_invoice else 'all_delivery'}} row row-cols-md-2 row-cols-lg-3 g-3 flex-nowrap flex-md-wrap mb32 all_billing_address">
```

Comment:
What's this class used for ? :thinking:

Reply:
the class was used in the previous implementation. it's not needed anymore so I have removed it.